masu/config.py

```diff
@@ -45,6 +45,10 @@ class Config(object):
 
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
+    # AMQP Message Broker
+    RABBITMQ_HOST = os.getenv('RABBITMQ_HOST', 'localhost')
+    RABBITMQ_PORT = os.getenv('RABBITMQ_PORT', '5672')
```

Comment:
No user or password is created. Using guest access for now, mostly because everything is on a private internal network that's inaccessible outside the pod.